src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -49,26 +49,30 @@ class AmendmentTableImpl : public AmendmentTable
     core::Clock::time_point m_firstReport; // close time of first majority report
     core::Clock::time_point m_lastReport;  // close time of most recent majority report
     beast::Journal m_journal;
+    AppApiFacade m_appApiFacade;
 
-    AmendmentState* getCreate (uint256 const& amendment, bool create);
+    AmendmentState& getCreate (uint256 const& amendment);
+    AmendmentState* getExisting (uint256 const& amendment);
     bool shouldEnable (std::uint32_t closeTime, const AmendmentState& fs);
     void setJson (Json::Value& v, const AmendmentState&);
 
 public:
-    AmendmentTableImpl (std::chrono::seconds majorityTime, int majorityFraction,
-            beast::Journal journal)
+    AmendmentTableImpl (
+        std::chrono::seconds majorityTime,
+        int majorityFraction,
+        beast::Journal journal)
         : m_majorityTime (majorityTime)
         , mMajorityFraction (majorityFraction)
         , m_firstReport (0)
         , m_lastReport (0)
-        , m_journal (journal)
+        , m_journal (std::move (journal))
```

Comment:
journals are lightweight and copyable, no need to move them or pass them by reference.